### PR TITLE
Another public Apple M1 Tech Preview

### PR DESCRIPTION
**Pull request by djs55**
### Proposed changes

Update the Apple M1 tech preview page with a new version.

### CODE REVIEW

**Comment by djs55** [comment]
I think this is ready. WDYT @usha-mandya @stephen-turner ?

File: docker-for-mac/apple-m1.md [hunk 1]
```diff
@@ -27,16 +27,20 @@ The tech preview of Docker Desktop for Apple M1 currently has the following limi
...
     In particular, the [mysql](https://hub.docker.com/_/mysql?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image is not available for ARM64. You can work around this issue by using a [mariadb](https://hub.docker.com/_/mariadb?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image.
 - The kernel may panic. If so, look in `~/Library/Containers/com.docker.docker/Data/vms/0/console.log` for a BUG or kernel panic to report.
-- The **Restart** option in the Docker menu may not work.
+
+## Fixes since the last preview
+
+- Kubernetes now works (although you might need to reset the cluster in our Troubleshoot menu one time to regenerate the certificates).
+- osxfs file sharing works.
+- The `host.docker.internal` and `vm.docker.internal` DNS entries now resolve.
+- We removed hard-coded IP addresses: it now dynamically discovers the IP allocated by macOS.
```
Comment (usha-mandya) [inline docker-for-mac/apple-m1.md:41]:
```suggestion
- Removed hard-coded IP addresses: Docker Desktop now dynamically discovers the IP allocated by macOS.
```

File: docker-for-mac/apple-m1.md [hunk 2]
```diff
@@ -27,16 +27,20 @@ The tech preview of Docker Desktop for Apple M1 currently has the following limi
...
-- Kubernetes does not initialize because of a missing DNS name.
-- osxfs file sharing does not work.
 - The HTTP proxy is not enabled.
 - Not all images are available for ARM64. You can add `--platform linux/amd64` to run an Intel image under emulation.
 
     In particular, the [mysql](https://hub.docker.com/_/mysql?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image is not available for ARM64. You can work around this issue by using a [mariadb](https://hub.docker.com/_/mariadb?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image.
 - The kernel may panic. If so, look in `~/Library/Containers/com.docker.docker/Data/vms/0/console.log` for a BUG or kernel panic to report.
-- The **Restart** option in the Docker menu may not work.
+
+## Fixes since the last preview
```
Comment (usha-mandya) [inline docker-for-mac/apple-m1.md:36]:
```suggestion
## Fixes since the Apple Silicon preview 7

2021-02-11
```

Comment (usha-mandya) [reply docker-for-mac/apple-m1.md:36]: Specified the preview version and the date so users can track changes

File: docker-for-mac/apple-m1.md [hunk 3]
```diff
@@ -27,16 +27,20 @@ The tech preview of Docker Desktop for Apple M1 currently has the following limi
...
+
+- Kubernetes now works (although you might need to reset the cluster in our Troubleshoot menu one time to regenerate the certificates).
+- osxfs file sharing works.
+- The `host.docker.internal` and `vm.docker.internal` DNS entries now resolve.
+- We removed hard-coded IP addresses: it now dynamically discovers the IP allocated by macOS.
+- We made a change that should improve disk performance.
```
Comment (usha-mandya) [inline docker-for-mac/apple-m1.md:42]:
```suggestion
- The updated version includes a  change that should improve disk performance.
```

**Comment by usha-mandya** [comment]
@djs55 Suggested some minor updates. PTAL

**Comment by djs55** [comment]
@usha-mandya thanks! I committed the suggestions.

**Review by usha-mandya — APPROVED** [review]
LGTM. Thanks @djs55

**Review by stephen-turner — APPROVED** [review]
LGTM

**Pull request merged**